constants_local.xml

```diff
@@ -9,6 +9,8 @@
         <CHAIN_ID>2</CHAIN_ID>
         <GENESIS_PUBKEY>02AAE728127EB5A30B07D798D5236251808AD2C8BA3F18B230449D0C938969B552</GENESIS_PUBKEY>
         <UPGRADE_TARGET_DS_NUM>1</UPGRADE_TARGET_DS_NUM>
+        <!-- Do not add a trailing "/" in the path-->
```

Comment:
@sandipbhoir Is this warning for users? Unfortunately, all the comments got stripped out when the file was processed before it's published on joining URLs. If this is important, we may need some checks in the code instead of a warning in the constants.xml